backend/app/routers/flows.py

```
@@ -0,0 +1,753 @@
+                config_id=flow.config_id,
+                content=plain_text,
+                extraction_result=new_data,
+                text_blocks=current_result.text_blocks,
+                schema_id=flow.schema_id,
+                schema_version=flow.schema_version,
+                result_id=new_result.id,
+                result_version=new_version,
+            )
+        except Exception:
+            # Don't fail the feedback submission if RAG storage fails
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)